qt/aboutdlg.py

```diff
@@ -80,9 +81,13 @@ def __init__(self, parent=None):
         vlayout.addLayout(hlayout)
 
     def _create_name_and_version_label(self):
```

Comment:
Here I fixed that old code. I am sure I have done this in the past. But the old stuff came back somehow.